addons/website/static/src/js/backend/res_config_settings.js

```diff
@@ -33,4 +44,36 @@ BaseSettingController.include({
     },
 });
 
+const WebsiteCookiesbarField = FieldBoolean.extend({
+    _onChange: function () {
+        const checked = this.$input[0].checked;
+        if (!checked) {
+            return this._setValue(checked);
+        }
+
+        const cancelCallback = () => {
+            return this.$input[0].checked = !checked;
+        };
+        Dialog.confirm(this, null, {
+            title: _t("Please confirm"),
+            $content: QWeb.render('website.res_config_settings.cookies_modal_main'),
+            buttons: [{
+                text: 'Do not activate',
+                classes: 'btn-primary',
+                close: true,
+                click: cancelCallback,
+            },
+            {
+                text: 'Activate anyway',
+                close: true,
+                click: () => {
+                    return this._setValue(checked);
+                },
```

Comment:
When your function is just a return, you can do `() => this._setValue(checked)` in one line. Same for cancelCallback ;)